x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecurityRestIT.java

```diff
@@ -34,24 +44,76 @@
 import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.equalTo;
 
-/**
- * This test suite will be run twice: Once against the fulfilling cluster, then again against the querying cluster. The typical usage is to
- * conditionalize on whether the test is running against the fulfilling or the querying cluster.
- */
 public class RemoteClusterSecurityRestIT extends ESRestTestCase {
     private static final String USER = "test_user";
     private static final SecureString PASS = new SecureString("x-pack-test-password".toCharArray());
     private static final String REMOTE_SEARCH_USER = "remote_search_user";
     private static final String REMOTE_SEARCH_ROLE = "remote_search";
 
+    private static LocalClusterConfigProvider commonClusterConfig = cluster -> cluster.module("analysis-common")
+        .feature(FeatureFlag.NEW_RCS_MODE)
+        .setting("xpack.license.self_generated.type", "trial")
+        .setting("xpack.security.enabled", "true")
+        .setting("xpack.security.authc.token.enabled", "true")
+        .setting("xpack.security.authc.api_key.enabled", "true")
+        .setting("xpack.security.http.ssl.enabled", "false")
+        .setting("xpack.security.transport.ssl.enabled", "true")
+        .setting("xpack.security.transport.ssl.key", "transport.key")
+        .setting("xpack.security.transport.ssl.certificate", "transport.crt")
+        .setting("xpack.security.transport.ssl.certificate_authorities", "transport-ca.crt")
+        .setting("xpack.security.transport.ssl.verification_mode", "certificate")
+        .keystore("bootstrap.password", "x-pack-test-password")
+        .keystore("xpack.security.transport.ssl.secure_key_passphrase", "transport-password")
+        .configFile("transport.key", Resource.fromClasspath("ssl/transport.key"))
+        .configFile("transport.crt", Resource.fromClasspath("ssl/transport.crt"))
+        .configFile("transport-ca.crt", Resource.fromClasspath("ssl/transport-ca.crt"))
+        .configFile("remote-cluster.key", Resource.fromClasspath("ssl/remote_cluster.key"))
+        .configFile("remote-cluster.crt", Resource.fromClasspath("ssl/remote_cluster.crt"))
+        .configFile("remote-cluster-ca.crt", Resource.fromClasspath("ssl/remote-cluster-ca.crt"))
+        .user(USER, PASS.toString());
+
+    public static ElasticsearchCluster fulfillingCluster = ElasticsearchCluster.local()
+        .name("fulfilling-cluster")
+        .nodes(1)
```

Comment:
This is a bit unnecessary. The default behavior is a single-node cluster although it's fine to keep this if you want to be extra explicit.

Reply:
That's good to know. I removed the explicit config.